job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -35,6 +35,11 @@ def SERVERLESS_LOGIC_DROOLS_CURRENT_PRODUCT_VERSION='8.26.0'
 def SERVERLESS_LOGIC_CURRENT_PRODUCT_BRANCH='main'
 def SERVERLESS_LOGIC_CURRENT_PRODUCT_CONFIG_BRANCH="master"
 
+def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='1.25.0'
+def SERVERLESS_LOGIC_DROOLS_NEXT_PRODUCT_VERSION='8.24.0'
```

Comment:
We should use the community versions:
```suggestion
def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='1.27.0'
def SERVERLESS_LOGIC_DROOLS_NEXT_PRODUCT_VERSION='8.27.0'
```

Reply:
@radtriste but next product version is 1.25.0, so isn't it correct as @lampajr did?

Reply:
@radtriste according to our private conversation, shouldn't the product version be 1.25... taking source code from 1.27 branch?